js/classes/entities/foreground/dialog/conversation.js

```diff
@@ -5,12 +5,12 @@
  */
 class Conversation {
     /**
-     * @param {Array<DialogBubble>} array The array containing all possible dialog bubbles in this conversation.
+     * @param {Array<DialogBubble|Choice>} array The array containing all possible dialog bubbles in this conversation.
```

Comment:
Why are you using a | and putting Choice on the right side of it? Is there a reason for that or could we just rename the array?

Reply:
Is this like a decision bubble?

Reply:
This is a typo - it should say DecisionBubble. The single bar means OR, in this context.